omegaconf/grammar/OmegaConfGrammarParser.g4

```diff
@@ -48,8 +48,9 @@ sequence: element (COMMA element)*;
 
 interpolation: interpolationNode | interpolationResolver;
 interpolationNode: INTER_OPEN DOT* configKey (DOT configKey)* INTER_CLOSE;
-interpolationResolver: INTER_OPEN (interpolation | ID) COLON sequence? BRACE_CLOSE;
+interpolationResolver: INTER_OPEN resolverName COLON sequence? BRACE_CLOSE;
 configKey: interpolation | ID | INTER_KEY;
+resolverName: (interpolation | ID) (DOT (interpolation | ID))* ;  // oc.env, myfunc, ns.${x}, ns1.ns2.f
```

Comment:
I had no idea we supported interpolations as function names here.
```
oc.env, myfunc, ns.${x}, ns1.ns2.f
```

```python
OmegaConf.register_resolver("aaa", lambda _ : "aaa" + _)
OmegaConf.register_resolver("bbb", lambda _ : "bbb" + _)
cfg = OmegaConf.create({"a": "${${b}:10}", "b": "aaa"})

cfg.a        # -> 'aaa10'

cfg.b = "bbb"

cfg.a        # -> 'bbb10'
```

Is this documented?

Reply:
> Is this documented?

No. It's kind of natural, but I guess it should be :)

Reply:
Also, I wasn't sure if it was worth documenting the namespace feature (since it will become obvious when `oc.env` is introduced), but I think I'll create a new doc entry that shows both namespace + interpolation in the resolver name.

Reply:
> I'll create a new doc entry that shows both namespace + interpolation in the resolver name.

Done in 1d82ae8

Reply:
I am considering this on the verge of too powerful, but willing to go with it because it's in already.
Do you have an actual use case?

Reply:
> Do you have an actual use case?

It's not critical since you can usually replace `${${func}:x}` with `${master:${func},x}` where `master` is a high-level resolver that dispatches calls to other functions based on its first argument. The advantage of a config-based version would be to not have to maintain this high-level resolver (also, it is easier if the resolvers are spread around the codebase and it's hard to access them from a single point, but that's probably a rare case).

Generally speaking, I can see use cases for dynamically choosing a different resolver, when there are multiple ways to compute a quantity and you want to be able to easily switch between them.